deploy/wip-catalog/templates/apiserver.yaml

```diff
@@ -68,7 +68,7 @@ spec:
     nodePort: {{ default 30001 .Values.insecureServicePort }}
     port: 80
     targetPort: {{ default 8081 .Values.insecurePort }}
-  {{- end }}
+  {{ end }}
```

Comment:
Why is that being removed? I think the result's going to be an extra line break, isn't it?